frontend/public/components/monitoring/_monitoring.scss

```diff
@@ -461,9 +461,11 @@ $monitoring-line-height: 18px;
   }
 }
 
-.query-browser__span-dropdown-menu {
-  right: 1px;
-  width: 130px;
+.query-browser__span-dropdown {
+  .pf-c-dropdown__menu {
```

Comment:
I get a little uneasy overriding internal PF styles as this could break on future versions. Can you share what these tweaks change and why they're needed?

Reply:
Thanks @spadgett. Good point. I have reworked this to not rely on the internal PatternFly class and use what I think is a cleaner approach.